webdictionary/src/test/java/com/eis0/kademlianetwork/IdFinderHandlerTest.java

```diff
@@ -0,0 +1,217 @@
+package com.eis0.kademlianetwork;
+
+import com.eis.smslibrary.SMSManager;
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.SMSPeer;
+import com.eis0.kademlia.DefaultConfiguration;
+import com.eis0.kademlia.KademliaId;
+import com.eis0.kademlia.SMSKademliaNode;
+import com.eis0.kademlia.SMSKademliaRoutingTable;
+import com.eis0.kademlianetwork.informationdeliverymanager.IdFinderHandler;
+import com.eis0.kademlianetwork.informationdeliverymanager.RequestTypes;
+import com.eis0.kademlianetwork.informationdeliverymanager.RequestsHandler;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({KademliaNetwork.class, SMSManager.class, KademliaJoinableNetwork.class})
+public class IdFinderHandlerTest {
+    //SEARCHER
+    private SMSPeer SEARCHER_PEER;
+    private KademliaId SEARCHER_ID;
+    private SMSKademliaNode SEARCHER;
+    //NODE1
+    private SMSPeer VALID_PEER1;
+    private KademliaId VALID_NODE1_ID;
+    private SMSKademliaNode VALID_NODE1;
+    //NODE2
+    private SMSPeer VALID_PEER2;
+    private KademliaId VALID_NODE2_ID;
+    private SMSKademliaNode VALID_NODE2;
```

Comment:
since they're not `static final` the shouldn't be in SCREAMING_CASE.
You can also setup most of these directly here, during declaration.

Reply:
You are right, I forgot to change the **SCREAMING_CASE** when I moved the initialization of the variables in the setup() method; I avoided the setup in the declaration to be sure that each time a test is started the nodes are built from zero, considering that in many cases some of their values are changed during a test (like the staleCount of a node)